src/dotgraph.cpp

```diff
@@ -103,6 +104,38 @@ static bool insertMapFile(TextStream &out,const QCString &mapFile,
 
 //--------------------------------------------------------------------
 
+DotAttributes::DotAttributes(QCString input)
+{
+  std::string tag = R"(\s*(\w+)\s*)";
+  std::string unquoted = R"(\s*([^\s,"]+)\s*)";
+  std::string quoted = R"(\s*\"([^"]*)\"\s*)";
```

Comment:
- How will a string like `"text1\"text2"` be handled?
- Is the special character `_` possible in the input string?

Reply:
unmatched text will be assigned to key `_unmatched_`

Reply:
underscore is possible.

Reply:
strings with quotes inside are not supported